tests/SideBySide/CancelTests.cs

```diff
@@ -228,6 +228,18 @@ public void DapperQueryMultiple()
 			}
 		}
 
+		[Fact]
+		public async Task CancelCompletedCommand()
+		{
+			using (var cmd = m_database.Connection.CreateCommand())
+			{
+				cmd.CommandText = @"select 1;";
+
+				using (await cmd.ExecuteReaderAsync().ConfigureAwait(false))
+					cmd.Cancel();
+			}
```

Comment:
Does this need to be followed by a second command that will be the unintentional victim of cancellation?